client/blocks/comments/form.scss

```diff
@@ -59,22 +59,18 @@
 	button {
 		opacity: 0;
 		position: absolute;
-		top: 4px;
-		right: 16px;
-		padding: 4px;
-		font-size: $font-body-extra-small;
-		font-weight: 600;
-		text-transform: uppercase;
-	color: var( --color-text-subtle );
+		float: right;
+		margin-top: 16px;
```

Comment:
Would it be possible to set `margin: 16px 0 -16px` here instead of setting margins in two places?

Reply:
Tried it out, this works better and cleans up the animation a bit.